### PR TITLE
misc/Shell.py: Add missing function

### LINKED ISSUE #3940: Missing function in coalib/misc/Shell.py
Fucntion call_without_output missing in shell.py 
Which is needed fot installation of cib
` File "/home/helios/coala_env/lib/python3.5/site-packages/coala_package_manager/installation_tool.py", line 29, in <module>
    from coalib.misc.Shell import call_without_output
ImportError: cannot import name 'call_without_output'
`

### PR DESCRIPTION
**Pull request by GPhantom**
Added call_without_output function.

Fixes #3940

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by meetmangukiya** [comment]
Add tests

**Comment by ashwini0529** [comment]
In case if you don't know where tests are written, go to https://github.com/coala/coala/blob/master/tests/misc/ShellTest.py

**Comment by adhikasp** [comment]
This is trying to solve the same issue as [this PR](https://github.com/coala/coala/pull/3942)

File: coalib/misc/Shell.py [hunk 1]
```diff
@@ -164,3 +164,7 @@ def get_shell_type():  # pragma: no cover
     if out.strip() == '$0':
         return 'cmd'
     return 'sh'
+
+
+def call_without_output(command):
+    run_shell_command(' '.join(command), stdout=PIPE, stderr=PIPE)
```
Comment (userzimmermann) [inline coalib/misc/Shell.py]: Better use `subprocess.DEVNULL` instead of `PIPE` here.

**Comment by userzimmermann** [comment]
@GPhantom Lost interest ? ;(

**Comment by jayvdb** [comment]
Ya, this is a dup, and I've closed the parent issue https://github.com/coala/coala/issues/3940 also.
Thanks for spotting this problem @adhikasp .

**Pull request closed without merging**